pl_bolts/models/self_supervised/byol/byol_module.py

```diff
@@ -155,30 +147,29 @@ def configure_optimizers(self):
         return [optimizer], [scheduler]
 
     @staticmethod
-    def add_model_specific_args(parent_parser):
+    def add_model_specific_args(parent_parser) -> ArgumentParser:
         parser = ArgumentParser(parents=[parent_parser], add_help=False)
         parser.add_argument("--online_ft", action="store_true", help="run online finetuner")
         parser.add_argument("--dataset", type=str, default="cifar10", choices=["cifar10", "imagenet2012", "stl10"])
 
         (args, _) = parser.parse_known_args()
 
-        # Data
+        # Data params
         parser.add_argument("--data_dir", type=str, default=".")
         parser.add_argument("--num_workers", default=8, type=int)
```

Comment:
I am fine with either. I think it might be useful to have these `add_dataset_specific_args` methods on datasets. If you decide to go with that route, then such PR would have to be merged before this one (and #873)

Reply:
I will add a `add_dataset_specific_args` method to the CIFAR10, STL10, ImageNet2012, and MNIST datamodules. I will try to open a PR in the next couple of days.